src/mixer/basetrackplayer.cpp

```diff
@@ -44,6 +45,15 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
     m_pChannel = new EngineDeck(channelGroup, pConfig, pMixingEngine,
                                 pEffectsManager, defaultOrientation);
 
+    m_pInputEnabled.reset(new ControlObjectSlave(group, "input_enabled", this));
+    m_pPassthroughEnabled.reset(new ControlObjectSlave(group, "passthrough", this));
+    m_pPassthroughEnabled->connectValueChanged(SLOT(slotPassthroughEnabled(double)));
```

Comment:
The only new CO in this PR is "input_enabled" which serves the same purpose as [MicN],enabled and [AuxN],enabled. I named it "input_enabled" since "enabled" would be less clear for the [ChannelN] namespace.

BTW passthrough is not vinyl control specific, though it is confusing that you must configure the "Vinyl Control N" input to get passthrough to work if you aren't using vinyl control.